pkg/config/config_evaluator.go

```diff
@@ -46,24 +46,43 @@ func NewConfigEvaluator(expireAt time.Duration, localConfigDir, repoConfigPath,
 	if err != nil {
 		return nil, fmt.Errorf("failed to create CEL environment: %w", err)
 	}
+	enableCaching := false
```

Comment:
Drive-by. I would prefer if we had a runtime config for "cache duration" and we could set it to something super small like "1s"

Reply:
@bradegler ^

Reply:
Yes. It will get moved into a command line option. I have that already teed up in a follow up.